code/streaming/Hacl.Streaming.MD.fst

```diff
@@ -70,12 +70,19 @@ open Hacl.Hash.Definitions
 module D = Hacl.Hash.Definitions
 module Agile = Spec.Agile.Hash
 
+let _: squash (inversion hash_alg) = allow_inversion hash_alg
+
+let is_supported = function
```

Comment:
No need for noextract on this?

Reply:
I think the definition is eliminated via bundling (and also happens to be Low*). I added it for good measure. Fixed in 2ce8329cd6 which is part of the subsequent PR.